.gitignore

```diff
@@ -104,3 +104,5 @@ e2e/tls
 mocks
 
 ui/.npm/
+
+.DS_Store
```

Comment:
what's this change about?

Reply:
To avoid committing the annoying MacOS Finder metadata files